tests/acceptance/settings-page/profile-test.js

```diff
@@ -0,0 +1,43 @@
+import profilePage from 'codecrafters-frontend/tests/pages/settings/profile-page';
+import userPage from 'codecrafters-frontend/tests/pages/user-page';
+import testScenario from 'codecrafters-frontend/mirage/scenarios/test';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'codecrafters-frontend/tests/helpers';
+import { signIn } from 'codecrafters-frontend/tests/support/authentication-helpers';
+
+module('Acceptance | settings-page | profile-test', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('can edit profile description', async function (assert) {
+    testScenario(this.server);
+    signIn(this.owner, this.server);
+
+    await profilePage.visit();
+    await profilePage.profileDescription.input.fillIn('Updated profile description');
+    await profilePage.profileDescription.input.blur();
+
+    await userPage.visit({ username: 'rohitpaulk' });
+    assert.strictEqual(
+      userPage.profileDescriptionMarkdown.text,
+      'Updated profile description',
+      'user page should reflect updated profile description',
+    );
+  });
+
+  test('tracks when the profile description is updated', async function (assert) {
```

Comment:
It's fine to add tests, but I think the analytics event itself should be tracked on the backend instead: https://github.com/codecrafters-io/core/pull/1264#discussion_r1543182307